frontend/src/js/controllers/dashCtrl.js

```diff
@@ -49,12 +52,14 @@
 
         vm.hostChallenge = function() {
 
-            var alert = $mdDialog.alert()
-                .title('Host a challenge')
-                .htmlContent('Please send an email to <a href="mailto:admin@cloudcv.org" class="blue-text">admin@cloudcv.org</a> with the details of the challenge and we will get back soon.')
-                .ok('Close');
-
-            $mdDialog.show(alert);
+            if ($rootScope.isAuth === true)
```

Comment:
Actually, I am not checking for accessing dashboard without login here, I only want to redirect the user according to its state like if he is not login then login page and challenge-host-team page. but on the other hand if he is logged in then directly to the challenge-host-team page. Please guide if I am wrong somewhere?